src/debug/shim/debugshim.cpp

```diff
@@ -210,14 +202,36 @@ STDMETHODIMP CLRDebuggingImpl::OpenVirtualProcess(
             }
         }
 
-        if(SUCCEEDED(hr))
+#ifdef FEATURE_PAL
+        if (SUCCEEDED(hr))
```

Comment:
Can we change the interface between DBI and dbgshim so that you pass the filename to DBI instead of a handle? It seems awkward to codify these PAL APIs as part of the official interface between the dlls.